cranelift/codegen/src/isa/s390x/inst.isle

```diff
@@ -3358,11 +3352,6 @@
 (rule (cond_br_bool (ProducesBool.ProducesBool producer cond) taken not_taken)
       (with_flags_side_effect producer (cond_br taken not_taken cond)))
 
-;; Emit a one-way conditional branch based on a boolean condition.
-(decl oneway_cond_br_bool (ProducesBool MachLabel) SideEffectNoResult)
-(rule (oneway_cond_br_bool (ProducesBool.ProducesBool producer cond) dest)
-      (with_flags_side_effect producer (oneway_cond_br dest cond)))
```

Comment:
Couldn't we have a `jt_sequence_bool` that implicitly decomposes the `ProducesBool` like the above?  Then we wouldn't have to introduce the `bool_producer` / `bool_cond` accessors just for this case ...

Reply:
Great idea, done!